src/content/1.7/basics/keeping_up-to-date/_index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Staying up-to-date
```

Comment:
Keep in mind that you can hava separate `title` (page title) and `menuTitle` (for the menu).

I think titles have to be carefully chosen to improve SEO.

I think we could title this one "How to keep PrestaShop up-to-date"

Reply:
Didn't know that, we will make use of this.